.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,45 @@
+name: Run pytest
+
+on:
+  push:
+    branches-ignore: 
+      - gh-pages
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        pip install pytest
+        pytest --html=../report/index.html
+    - name: Upload report
+      env:
+        GH_PAT: ${{ secrets.GH_PAT }}
```

Comment:
I'm a *little* leery of using a PAT here. A deploy key seems like it would be a safer choice since it would be repo-specific (as opposed to a PAT, which AFAICT gives r/w access to all my public repos). Something like this looks workable: https://github.community/t5/GitHub-Actions/Github-Actions-CI-how-to-use-store-deploy-key-to-download-from/m-p/30251/highlight/true#M351

Reply:
Well then, what email address should I use?